packages/widgets/src/Pagination/PageControls.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import { Button } from '@dhis2/ui-core'
+import { ChevronRight, ChevronLeft } from '@dhis2/ui-icons'
+import { spacers } from '@dhis2/ui-constants'
+
+import translate from '../translate'
+
+const PageControls = ({
+    dataTest,
+    onClick,
+    nextPageText,
+    page,
+    pageCount,
+    previousPageText,
+}) => (
+    <div data-test={`${dataTest}-pagecontrols`}>
```

Comment:
I think we should use `secondary` type buttons for the 'Previous' and 'Next' buttons.